taxonomy/management/commands/refresh_job_skills.py

```diff
@@ -93,8 +76,4 @@ def handle(self, *args, **options):
         """
         Entry point for management command execution.
         """
-        ranking_facet, nested_ranking_facet = options['ranking_facet'], options['nested_ranking_facet']
-        if not ranking_facet and nested_ranking_facet:
-            raise CommandError(_('Error: the following arguments are required: ranking_facet, nested_ranking_facet'))
-
-        self._refresh_jobs(ranking_facet, nested_ranking_facet)
+        self._refresh_jobs(RankingFacet.TITLE_NAME, RankingFacet.SKILLS)
```

Comment:
Any reason why we are removing ranking facets from management command?

Reply:
[This Line](https://github.com/edx/taxonomy-connector/pull/36/files#diff-be1337b715bfbfcab2ac7d9ace6c2173d7d9d3bf60a5505dfc4c0842337718daR34) expects `Skill`'s `external_id` to be returned by `skill_data['name']` which is only possible if  `nestedRankingFacet` is `RankingFacet.SKILLS`. If this is not the case code will break.

 It is not a good idea for the code to depend on its input and since we won't be calling the management command with any input other then `RankingFacet.TITLE_NAME` and `RankingFacet.SKILLS`, I (after discussing with Zaman and Irfan) decided it would be better to hard code the ranking facets.